docs/docs/installation/configuration/argo.md

```diff
@@ -0,0 +1,45 @@
+---
+comments: true
+---
+
+# Deploy Keptn via ArgoCD
+
+Keptn can be deployed on your Kubernetes cluster
+via [ArgoCD](https://argo-cd.readthedocs.io/en/stable/).
+To be able to do that, you need to have ArgoCD installed
+on your cluster.
+You can find the
+[installation instructions](https://argo-cd.readthedocs.io/en/stable/operator-manual/installation/)
+in the ArgoCD documentation.
```

Comment:
I think instead of this, we should have a "before you begin" section at the top that says that we assume argo is installed already